tools/Makefile.win

```diff
@@ -392,17 +392,22 @@ ifeq ($(CONFIG_BUILD_2PASS),y)
 	$(Q) $(MAKE) -C $(CONFIG_PASS1_BUILDIR) LINKLIBS="$(LINKLIBS)" USERLIBS="$(USERLIBS)" "$(CONFIG_PASS1_TARGET)"
 endif
 	$(Q) $(MAKE) -C $(ARCH_SRC) EXTRA_OBJS="$(EXTRA_OBJS)" LINKLIBS="$(LINKLIBS)" EXTRAFLAGS="$(KDEFINE) $(EXTRAFLAGS)" $(BIN)
+	echo $(BIN) > $(NUTTXNAME).manifest
+	printf '%s\n' *.map >> $(NUTTXNAME).manifest
```

Comment:
can we use echo "*.map"?

Reply:
echo will put them all on the same line so it would look like this:

```
nuttx
nuttx.hex
System.map User.map
```

I think printf is the best option